constants.xml

```diff
@@ -4,7 +4,7 @@
     <general>
         <DEBUG_LEVEL>3</DEBUG_LEVEL>
         <ENABLE_DO_REJOIN>false</ENABLE_DO_REJOIN>
-        <LOOKUP_NODE_MODE>false</LOOKUP_NODE_MODE>
+        <LOOKUP_NODE_MODE>true</LOOKUP_NODE_MODE>
```

Comment:
This is an error is assume, you didnt mean to change this in the master

Reply:
nope